### ISSUE TITLE
npm7 and react 17 peer dependency error

### ISSUE THREAD

**Issue by tiagoporto** [hodgef/react-simple-keyboard#980]
Npm 7 [Automatically installing peer dependencies](https://github.com/npm/rfcs/blob/latest/implemented/0025-install-peer-deps.md) and the last version of react is 17.

When using react 17 with npm 7 I receive the following error

![Screen Shot 2021-01-13 at 5 12 55 PM](https://user-images.githubusercontent.com/3688905/104505630-f5282f80-55c2-11eb-8dd1-a86bfa6f7aba.png)

Fix

```json
"react": "^16.14.0 | ^17.0.0",
"react-dom": "^16.14.0 | ^17.0.0"
```

**Comment by hodgef** [comment]
Hello @tiagoporto,

This library uses Enzyme for its tests, and support has [not yet landed for react 17](https://github.com/enzymejs/enzyme/pull/2430). Also npm@latest and npm@next still point to version 6.x.

For those reasons I will hold off on upgrading for now. That being said, as soon as the official Enzyme adapter is released I will upgrade and post in this ticket. In the meantime, I'll keep this open and flagged as a bug.

Regards,
Francisco Hodge

**Comment by tiagoporto** [comment]
Hi @hodgef, just an update, [npm 7 is the official ](https://github.blog/2021-02-02-npm-7-is-now-generally-available/).

**Comment by hodgef** [comment]
Hey @tiagoporto, I've just merged the React version updates (per #1010). Sorry for the delay!

Regards,
Francisco Hodge

**Comment by tiagoporto** [comment]
Thank you @hodgef
